--- /etc/hostapd/hostapd.conf.orig	2017-08-18 13:50:07.981939023 +0000
+++ /etc/hostapd/hostapd.conf	2017-08-18 14:25:14.780338112 +0000
@@ -80,7 +80,7 @@
 ##### IEEE 802.11 related configuration #######################################
 
 # SSID to be used in IEEE 802.11 management frames
-ssid=test
+ssid=raspberrypi
 # Alternative formats for configuring SSID
 # (double quoted string, hexdump, printf-escaped string)
 #ssid2="test"
